tools/cockpit.spec

```diff
@@ -149,7 +149,7 @@ The Cockpit Web Service listens on the network, and authenticates users.
 env NOCONFIGURE=1 ./autogen.sh
 %endif
 %configure --disable-static --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding}
-make -j %{?extra_flags} all
+make %{?extra_flags} all
```

Comment:
We would need to give more CPUs to the VM as well.  Performance in general has suffered, of course...